### ISSUE TITLE
read c++ name mangling

### ISSUE THREAD

**Issue by GoogleCodeExporter** [eteran/edb-debugger#108]
```
On a large project, with no debug information, but symbol tables (as read by 
there is a function _Z6foobarijPhi as reported by readelf -s libwhatever.so. 

gdb correctly shows the function as foobar(int, unsigned int, unsigned char*, 
int) in the backtrace/frame information, but edb doesn't appear to try 
displaying it, or any other function, at all.
The "Find function" plugin also doesn't list any function in the libwhatever.so 
range.

in the 0.9.17 deb version
```

Original issue reported on code.google.com by `evan.teran` on 3 Oct 2012 at 3:22

**Comment by 10110111** [comment]
Fixed.

**Comment by tehrengruber** [comment]
So I tried using c++filt to manually postprocess the generated .map files of my binary. But afterwards I have only very few symbols of the Binary left in the SymbolFinder. Did someone actually try this?

Some further research showed that the address of the main routine is not even found anymore, but manually looking into the demangled .map file the line containing the corresponding address was the same as before.

Is there any reference how the .map files have to look, such that they are parsed properly. For example I observed that the demangled map file contains whitespaces. Is this the problem? And if so, shouldn't whitespaces be supported?
